xapian-core/cluster/cluster.cc

```diff
@@ -110,18 +121,16 @@ TermListGroup::add_document(const Document &document) {
 }
 
 void
-TermListGroup::add_documents(DocumentSource &docs) {
+TermListGroup::add_documents(MSetDocumentSource docs) {
```

Comment:
Do we need `MSetDocumentSource` at all? We don't have any other types of `DocumentSource` at the moment, so we could just pass an `MSet` or a pair of `MSetIterator`s in here.